base_local_planner/src/simple_trajectory_generator.cpp

```diff
@@ -187,7 +187,7 @@ bool SimpleTrajectoryGenerator::generateTrajectory(
       base_local_planner::Trajectory& traj) {
   double vmag = hypot(sample_target_vel[0], sample_target_vel[1]);
   double eps = 1e-4;
```

Comment:
since this is a constant we should either make it static const or define it outside of the class as the same since we use it in a few places.